lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -32,41 +29,36 @@ const vehicleSymbols = [
 function VehicleTooltip (props) {
   const { direction, intl, permanent, vehicle } = props
 
-  let name = vehicle?.label
-  if (name !== null && name?.length <= 5) {
-    const mode = vehicle.routeType
-      ? intl.formatMessage({
-        id: `common.otpTransitModes.${vehicle.routeType.toLowerCase()}`
-      })
-      : intl.formatMessage({
-        id: `components.TransitVehicleOverlay.line`
-      })
-    name = intl.formatMessage(
-      { id: 'components.TransitVehicleOverlay.realtimeVehicleName' },
-      // Only render space if name is present
-      { headsign: `${name && ' '}${name}`, mode }
+  let vehicleLabel = vehicle?.label
+  if (vehicleLabel !== null && vehicleLabel?.length <= 5) {
```

Comment:
Add a comment regarding the assumption being made regarding `vehicleLabel`.

Reply:
Added in 
092b53f